css/exihib-statment.css

```diff
@@ -18,6 +18,33 @@ ul#list-container li:hover {
   opacity: 0.6;
 }
 
+@media only screen and (max-width: 500px) {
+  .l-list1 {
+    margin: 0 20px 0 0;
+  }
+  .ex-statement-p,
+  a {
+    font-family: Poppins;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 22px;
+    line-height: 33px;
+  }
+
+  #title-ex {
+    font-size: 26px;
+  }
+}
+@media only screen and (max-width: 1200px) {
+  .ex-statement-p,
+  a {
```

Comment:
and the same here please, if you take out the comma that is enough - without the comma, the style will target a elements inside the class `ex-statement-p` but with the comma you are targeting the class  `ex-statement-p`  AND  a elements.

Reply:
Ok thanks, I removed the comma.